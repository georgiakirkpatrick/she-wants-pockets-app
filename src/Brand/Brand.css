.Brand {
    height: 250px;
    width: 160px;
}

.Brand img {
    height: 75%;
}

.Brand p {
    font-size: 16px;
}