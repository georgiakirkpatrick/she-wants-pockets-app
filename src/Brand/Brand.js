import React from 'react';
import './Brand.css';

function Brand(props) {
    return (
        <a href={props.brandLink}>
            <div className='Brand'>
                <img src={props.pathToImage} alt={props.imgAlt} />
                <p>{props.brandName}</p>
            </div>
        </a>
    )
}

export default Brand;