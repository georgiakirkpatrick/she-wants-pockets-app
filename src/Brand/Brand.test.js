import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Brand from './Brand'

describe ('Brand', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<Brand />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it ('renders the UI as expected', () => {
        const tree = renderer
            .create(<Brand />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})