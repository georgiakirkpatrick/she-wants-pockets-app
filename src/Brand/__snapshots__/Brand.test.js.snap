// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand renders the UI as expected 1`] = `
<a>
  <div
    className="Brand"
  >
    <img />
    <p />
  </div>
</a>
`;
